pipelines/templates/bosh-lite-cf-edge.yml

```diff
@@ -123,7 +123,7 @@ jobs:
             BOSH_TARGET: <%= full_deployment_name +  '.' + domain_name %>
             BOSH_LITE_NAME: {{deployment-name}}
             RUBYGEM_MIRROR: {{rubygem-mirror}}
-            DOMAIN_NAME: {{domain_name}}
+            DOMAIN_NAME: {{domain-name}}
```

Comment:
So where is this `domain-name` variable within the scope of the pipeline coming from? I thought that the value of `full_config["domain-name"]` is present in the pipeline variable scope as `domain_name` and not as `domain-name`?